src/recipient_dashboard/templates/recipient_dashboard/dashboard.html

```diff
@@ -1,90 +1,137 @@
+                    </div>
+                </div>
             </div>
-        {% endif %}
-
-        <h1 class="title">Welcome, {{ user.first_name }} {{ user.last_name }}!</h1>
-        <h2 class="subtitle mt-3">List of Available Donations:</h2>
-        {% if donations %}
-            <div class="columns is-multiline is-variable is-4">
-                {% for donation in donations %}
-                    <div class="column is-full-mobile is-half-tablet is-half-desktop">
-                        <div class="card mb-5">
-                            <div class="card-content">
-                                <div class="media is-flex is-justify-content-space-between is-flex-direction-column-mobile">
-                                    <div class="media-content">
-                                        <p class="title is-4 is-clipped">{{ donation.food_item }}</p>
-                                    </div>
-                                    <div class="media-right mt-2-mobile">
-                                        <span class="tag is-primary is-light is-medium mt-1">{{ donation.quantity }} Available</span>
-                                    </div>
-                                </div>
-                                <div class="content">
-                                    <div class="columns is-flex-direction-column-mobile">
-                                        <div class="column is-two-thirds">
-                                            <a class="button is-link is-light is-fullwidth"
-                                               style="white-space: normal;
+            <div class="column">
+                <div class="field">
+                    <div class="control">
+                        <span class="select">
+                            <select name="category">
+                                <option value="">All Categories</option>
+                                {% for cat in categories %}
+                                <option value="{{ cat }}" {% if category == cat %}selected{% endif %}>{{ cat }}</option>
+                                {% endfor %}
+                            </select>
+                        </span>
+                    </div>
+                </div>
+            </div>
+            <div class="column">
+                <div class="field is-grouped">
+                    <div class="control">
+                        <button class="button is-info" type="submit">
+                            Search
+                        </button>
+                    </div>
+                    <div class="control">
+                        <a href="{% url 'recipient_dashboard' %}" class="button is-danger">
+                            Clear all
+                        </a>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </form>
+    {% if donations %}
+    <div class="columns is-multiline is-variable is-4">
+        {% for donation in donations %}
```

Comment:
Can we have a check to display only those donations whose quantity is greater than 0. Currently the donations with 0 quantity are also being displayed? 
Thanks Estella!

Reply:
Done! Also added a field where users can filter for donations with a quantity greater than or equal to a certain value :)

Reply:
Awesome work Estella!